### PR TITLE
Changed active-background colors

### PR DESCRIPTION
**Pull request by ShimiSun**

### CODE REVIEW

File: src/js/index.js [hunk 1]
```diff
@@ -54,7 +54,10 @@ export const hpe = deepFreeze({
         dark: '#444444',
       },
       control: 'brand',
-      'active-background': 'background-contrast',
+      'active-background': {
+        dark: '#FFFFFF1F',
+        light: '#CCCCCC99',
```
Comment (taysea) [inline src/js/index.js]: We want 99? The design has 60%

Comment (ShimiSun) [reply src/js/index.js]: 60% == 99. 
99 doesn't mean 99% cause letters are also part of the calculation.
Take a look at this resource:
https://gist.github.com/lopspower/03fb1cc0ac9f32ef38f4

Comment (taysea) [reply src/js/index.js]: Oh nice! I didn't know that.

Comment (gregfuruiye) [reply src/js/index.js]: Yes, so the hexidecimal code for the transparency is different from the actual value we are using in figma. I'm using this resource to specify the transparency for hex colors: https://gist.github.com/lopspower/03fb1cc0ac9f32ef38f4

Comment (gregfuruiye) [reply src/js/index.js]: When we use transparency in our color styles for the design system, we can work to provide the hex number to include the transparency value as well.

**Pull request merged**